### DISCUSSION TITLE
RPM and Generic Gauge Field

### DISCUSSION THREAD

**Discussion by Wings64** [nguyenquyhy/Flight-Tracker-StreamDeck#207] [Q&A] [answered]
Is there a parameter for propeller RPM? Engine RPM is not a useful substitute, but is all that seems to be available on lists I have found .

Four places of decimals are needed to resolve the full resolution of the Caravan pitch trim. What is the function of the Unit field next to the Decimals in the Generic Gauge? Can this be used to adjust displayed digits?

**Comment by nguyenquyhy** [comment, marked as answer]
Have you tried `PROP RPM:index`?

I don't really understand the second part of your question. Are you asking about this?
![image](https://user-images.githubusercontent.com/1560418/110630394-4a239480-8173-11eb-99ef-dbb6dd3edf7f.png)

**Comment by Wings64** [comment]
Hi,

 

Thanks for responding.

 

Yes. My unit question was about that ‘Empty for default unit’ field.

 

However, I have a bigger problem! Since the latest FS2020 update to 1.14.5.0 I have lost much of the Flight Tracker functionality despite doing a full reinstall of your plugin and a full computer reboot as well. Controls seem OK still, but none of the feedback parameters are now displayed which all worked OK before. As an example, I can select Caravan pitot heating, but get no feedback. Similarly for other such controls. Fordy panel and the Caravan panel from the web affected as well as my own.

 

I’m running the latest Steam Deck software 4.9.3.13222 under Win10 x60 Pro.

 

PROP_RPM is not on any of my parameter lists, but I can’t test it because of the above problem.

 

Where can I go from here please?

 

Thanks!

 

David

**Comment by Wings64** [comment]
Further to my last, I’m pleased to say that a Repair of FS2020 has restored normal operation of your Stream Deck gauges.

 

But PROP_RPM does not generate a response.

 

Thanks as always for your invaluable work.

 

Regards,

 

David

 

____________________________________________________________________________

Hi,

 

Thanks for responding.

 

Yes. My unit question was about that ‘Empty for default unit’ field.

 

However, I have a bigger problem! Since the latest FS2020 update to 1.14.5.0 I have lost much of the Flight Tracker functionality despite doing a full reinstall of your plugin and a full computer reboot as well. Controls seem OK still, but none of the feedback parameters are now displayed which all worked OK before. As an example, I can select Caravan pitot heating, but get no feedback. Similarly for other such controls. Fordy panel and the Caravan panel from the web affected as well as my own.

 

I’m running the latest Steam Deck software 4.9.3.13222 under Win10 x60 Pro.

 

PROP_RPM is not on any of my parameter lists, but I can’t test it because of the above problem.

 

Where can I go from here please?

 

Thanks!

 

David

**Comment by Wings64** [comment]
OK. I got there at last. The parameter is PROP_RPM__1 (with final double underscore) which I got from your latest parameter list – not there in the earlier version I had.

 

Thanks again!

 

David

 

________________________________________________________________________________

 

Further to my last, I’m pleased to say that a Repair of FS2020 has restored normal operation of your Stream Deck gauges.

 

But PROP_RPM does not generate a response.

 

Thanks as always for your invaluable work.

 

Regards,

 

David

 

____________________________________________________________________________

Hi,

 

Thanks for responding.

 

Yes. My unit question was about that ‘Empty for default unit’ field.

 

However, I have a bigger problem! Since the latest FS2020 update to 1.14.5.0 I have lost much of the Flight Tracker functionality despite doing a full reinstall of your plugin and a full computer reboot as well. Controls seem OK still, but none of the feedback parameters are now displayed which all worked OK before. As an example, I can select Caravan pitot heating, but get no feedback. Similarly for other such controls. Fordy panel and the Caravan panel from the web affected as well as my own.

 

I’m running the latest Steam Deck software 4.9.3.13222 under Win10 x60 Pro.

 

PROP_RPM is not on any of my parameter lists, but I can’t test it because of the above problem.

 

Where can I go from here please?

 

Thanks!

 

David
